docs/HowTo/Introduction.md

```diff
@@ -0,0 +1,118 @@
+most applications. However, we understand that what may be low impact for one scenario may be very impactful breaking change
+for another. For details, please see [here](BreakingChanges.md).
+
+# Usage
+
+The tools are available as a command line tool, a VS plugin, as well as a collection of libraries. The VS plugin is updated at
+a slower pace than the command line tool (for instance, it currently does not support the breaking change functionality). The
+command line tool is available at [github](http://github.com/microsoft/dotnet-apiport/releases), and requires .NET 4.6 or Mono 
+(there is a [work item](https://github.com/Microsoft/dotnet-apiport/issues/117) to convert this to a DNX application).
+
+The tool understands three commands:
+
+### Analyze
+
+Arguably the most important function of the tool is its ability to analyze an assembly. This can take in a file, collection of
+files, or a directory of assemblies.  
+
+`ApiPort.exe analyze <options>`
+
+The current options are:
+
+```
+  -f, --file=VALUE           [Required] Path to assembly file or directory of
+                               assemblies.
+  -o, --out=VALUE            Output file name
+  -d, --description=VALUE    Description of the submission
+  -t, --target=VALUE         The target you want to check against.
+  -r, --resultFormat=VALUE   The report output format
+  -p, --showNonPortableApis  Calculate non-portable APIs
+  -b, --showBreakingChanges  Calculate breaking changes on full .NET Framework
+      --noDefaultIgnoreFile  Do not use the standard assembly ignore list
+                               when analyzing breaking changes. The default
+                               ignore list can be found at KnownSafeBreaks.json
+  -i, --ignoreAssemblyFile=VALUE
+                             Specifies a json file defining assemblies that
+                               should not be analyzed for specific targets
+                               while analyzing breaking changes. This can be
+                               useful for excluding assemblies that are known
+                               to not regress on certain .NET Framework
+                               versions due to breaking changes. Note that,
+                               currently, this parameter only affects breaking
+                               change analysis; not portability analysis.
+  -s, --suppressBreakingChange=VALUE
+                             Specifies a breaking change (by ID) to suppress
+                               during breaking change analysis. Any breaking
+                               changes with IDs specified for suppression will
+                               not be reported.
+  -h, -?, --help             Show help
+```
+
+For more details on analysis for portability, look [here](PlatformPortability.md) and for breaking change analysis, please
+look [here](BreakingChanges.md).
+
```

Comment:
I think some sample command lines would be helpful (especially some involving -t since it's likely the least obvious parameter). I know there are a couple in the other documents, but I think a few more could help.